### DISCUSSION TITLE
怎么显示角色对应的皮肤头像

### DISCUSSION THREAD

**Discussion by Tayeusym** [bs-community/blessing-skin-server#632] [Q&A] [answered]
我开自己的地铁MC服务器，用着MTR模组。我想改模组中网页地图中显示玩家头像的部分（如下图）。
![QQ截图20240608185644](https://github.com/bs-community/blessing-skin-server/assets/147396405/fd109429-3c43-4956-87e2-2f1aafe6aa0a)
这个模组显示头像的默认使用的是minotar.net获取的（即正版），代码为：
https://minotar.net/avatar/${player}
（其中“${player}”指显示对应的玩家名）。
因为服务器使用的是authlib-injector外置登录模式，认证服务器为我的皮肤站，这样就导致网页上有部分个别玩家的头像显示的是正版的而不是皮肤站中设置的角色的（个别玩家用的角色名是与正版一致所以会显示头像，没有的则会显示史蒂夫头像）。我想把他改成我皮肤站中获取的玩家角色头像，所以代码修改为：
https://camildoc.top/avatar/${player}
但我试过用浏览器访问这个页面，将“${player}”改成我的角色名，结果并未返回正确的角色所用的皮肤头像，而是显示默认的史蒂夫头像。
请问怎么实现使用角色名显示对应的角色所用的皮肤的头像？

**Comment by tnqzh123** [comment, marked as answer]
目前 Blessing Skin 没有通过角色名获取头像的功能，建议自行编写插件实现。
